### DISCUSSION TITLE
Trang trí báo cáo nè he

### DISCUSSION THREAD

**Discussion by thaopham2501** [baonhu1304/Lop-10A2-nhom-1#27] [Ideas]
chốt 1 cái là concept noel vì mình ế, cái còn lại ai có ý tưởng lên tiếng đi

**Comment by baonhu1304** [comment]
Hay mình làm chung 1 concept đi. T làm phụ cho nghen

**Comment by baonhu1304** [comment]
@thaopham2501 cho xin cái link bài báo cáo i nghen

**Reply by thaopham2501** [reply]
chưa bắt đầu làm nữa :v chill bruh, tối nay gửi

**Reply by baonhu1304** [reply]
oke iu iu
